### PR TITLE
update facebook grant

### PR DESCRIPTION
**Pull request by festinalente91**
- updated facebook user grant to only required facebook token to grant  again.

### CODE REVIEW

File: src/server/auth/controllers.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default {
       break;
     case GRANT_TYPE.FACEBOOK:
       const {facebookToken} = req.body;
-      grant = AuthModel.grantFacebook(userKey, facebookToken);
```
Comment (ochanje210) [inline src/server/auth/controllers.js:38]: Awesome!

Comment (festinalente91) [reply src/server/auth/controllers.js:38]: done!

**Review by ovekyc — APPROVED** [review]

**Comment by ovekyc** [comment]
LGTM

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**